Core/Entities/Comment.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Core.Entities
+{
+    public class Comment : EntityBase
+    {
+        public Guid UserId { get; set; }
+        public User User { get; set; }
+        public string Message { get; set; }
+        public Guid PostId { get; set; }
+        public Post Post { get; set; }
```

Comment:
why you have id and user id, post and postId in one place?

Reply:
it is rule for creating relationship between entities//tables

Reply:
Why this relationship is created inside a comment?